mlrun/launcher/client/local.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 MLRun Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from mlrun.launcher.abstract import AbstractLauncher
+
+
+class ClientLocalLauncher(AbstractLauncher):
```

Comment:
Id expect the `mlrun/launcher` source tree to be like:

mlrun/launcher/factory.py
mlrun/aluncher/base.py
mlrun/aluncher/local.py
mlrun/aluncher/remote.py

class, w.r.t class names

base.py
`_BaseLauncher()`

local.py
`LocalLauncher()`

Reply:
by that, you will remove many files/dirs that are overwhelming for now. also, keep the functions stubs to minimal, let them grow upon need / tasks